mypy/build.py

```diff
@@ -2559,14 +2559,30 @@ def log_configuration(manager: BuildManager) -> None:
     """Output useful configuration information to LOG and TRACE"""
 
     manager.log()
-    configuration_vars = (
+    configuration_vars = [
         ("Mypy Version", __version__),
         ("Config File", (manager.options.config_file or "Default")),
-        ("Configured Executable", manager.options.python_executable),
+    ]
+
+    src_pth_str = "Source Path File"
+    src_pths = manager.source_set.source_paths
+
+    if len(src_pths) > 1:
+        src_pth_str += "s"
+        configuration_vars.append((src_pth_str, ""))
+        for source_path in src_pths:
+            configuration_vars.append(("." * len(src_pth_str), source_path))
```

Comment:
This looks odd to me. For consistency with other verbose, I think that it would be better to show all source paths on a single line, separated by spaces.